docs/concepts/trading-on-vega/market-lifecycle.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_position: 7
+title: Market lifecycle
+hide_title: false
+description: See every stage possible for a proposed or live market.
+---
+The market lifecycle begins when a proposal for a new market is accepted, and is driven by the market's state. The table shows a high-level view of each part of a market's lifecycle, including when that stage is over and a new one begins. Below, learn more about each lifecycle stage.
```

Comment:
**thought (non-blocking)**:  This page is great as is, but I think could be clearer with a diagram. There's one in the [spec](https://github.com/vegaprotocol/specs/blob/master/protocol/0043-MKTL-market_lifecycle.md), but that's out of date. Perhaps worth a follow up task?